learning/tour-of-beam/frontend/lib/pages/tour/state.dart

```diff
@@ -37,7 +37,7 @@ import 'path.dart';
 
 class TourNotifier extends ChangeNotifier with PageStateMixin<void> {
   final ContentTreeController contentTreeController;
-  final PlaygroundController playgroundController;
+  PlaygroundController? playgroundController;
```

Comment:
TourNotifier must always have a `final PlaygroundController`.
When I talked nullable, I meant that `TobScaffold.playgroundController` should be `null`.